01-js-intro-and-jquery/other/1_12/resources/script/script.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+var onClickButton =  function () {
```

Comment:
This function name should be onButtonClick you are not buttoning a click :P hahahaha